<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>easyForm</class>
 <widget class="QWidget" name="easyForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="25" column="1">
    <widget class="QLineEdit" name="sumLitEdit">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="25" column="0">
    <widget class="QLabel" name="label_17">
     <property name="text">
      <string>Сумма прописью</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>ДЕБЕТ</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLineEdit" name="receiverNumEdit">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="18" column="1">
    <widget class="QLineEdit" name="bankTrEdit">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Счет №</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="19" column="1">
    <widget class="QLineEdit" name="bankTrBIKEdit">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="4" colspan="2">
    <widget class="QLineEdit" name="sumDigEdit1"/>
   </item>
   <item row="14" column="1">
    <widget class="QLineEdit" name="bankNumEdit">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Получатель</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>ИНН</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QLabel" name="label_20">
     <property name="text">
      <string>В т.ч. символами</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDateEdit" name="dateEdit"/>
   </item>
   <item row="12" column="4" colspan="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Заполнить</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>№ Ордера</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QLineEdit" name="INNEdit">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Дата</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4" rowspan="11" colspan="2">
    <widget class="QTableWidget" name="tableWidget">
     <property name="columnCount">
      <number>2</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>127</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>Новая строка</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Новая строка</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Новая строка</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Символ</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Сумма</string>
      </property>
     </column>
     <item row="0" column="1">
      <property name="text">
       <string>1</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="orderNumEdit">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="27" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="26" column="0">
    <widget class="QLabel" name="label_18">
     <property name="text">
      <string>Источник поступления</string>
     </property>
    </widget>
   </item>
   <item row="26" column="1">
    <widget class="QLineEdit" name="incomeEdit">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>КРЕДИТ</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="fromEdit">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="23" column="1">
    <widget class="QLineEdit" name="bankRecBIKEDit">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Счет №</string>
     </property>
    </widget>
   </item>
   <item row="18" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Наименование</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="27">
    <widget class="Line" name="line_7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="21" column="0" colspan="2">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>БАНК-ПОЛУЧИТЕЛЬ</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="16" column="0" colspan="2">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>БАНК-ВНОСИТЕЛЬ</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QLabel" name="label_19">
     <property name="text">
      <string>Сумма цифрами</string>
     </property>
    </widget>
   </item>
   <item row="24" column="1">
    <widget class="Line" name="line_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="23" column="0">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>БИК</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="fromNumEdit">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Счет №</string>
     </property>
    </widget>
   </item>
   <item row="22" column="0">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>Наименование</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>От кого</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLineEdit" name="receiverEdit">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="19" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>БИК</string>
     </property>
    </widget>
   </item>
   <item row="20" column="1">
    <widget class="Line" name="line_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="22" column="1">
    <widget class="QLineEdit" name="bankRecEdit">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="13" column="4" colspan="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Загрузить из базы</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
